lib/util.js

```diff
@@ -410,7 +394,7 @@ function formatValue(ctx, value, recurseTimes) {
     });
   }
 
-  var constructor = getConstructorOf(value);
+  var constructor = internalUtil.getConstructorOf(value);
```

Comment:
Btw, `internal/util` has now been refactored to use the `module.exports = { ... }` format.